cg/src/domains/management/management.routes.js

```diff
@@ -146,6 +151,17 @@ module.exports = app => {
     asyncHandler(async (req, res) => usersController.updatePassword(req, res))
   );
 
+  router.put(
```

Comment:
We're mixing REST here, it makes no sense to start doing it now. Either we change everything or keep it as it was before. Please change these endpoints

Reply:
Well, to some extent this makes sense, although I do not like and idea to do something wrong just to be consistent with something.